src/main.cpp

```diff
@@ -2068,6 +2068,13 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY;
     }
 
+    bool fBIP102Enforcing = (pindex->GetBlockTime() >= (int64_t)BIP102_FORK_TIME);
+    if (fBIP102Enforcing && block.nVersion >= 5 && !IsSuperMajority(5, pindex->pprev, chainparams.GetConsensus().nMajorityRejectBlockOutdated, chainparams.GetConsensus()))
```

Comment:
If version is lower than 5 it doesn't matter the result of the IsSuperMajority call?
You don't need an if and to assign false to the bool: just assign all the conditions to the variable directly . It's simpler and you would have probably not made the mistake you did if it wasn't for the unnecessary if statement.